x-pack/plugins/security_solution/public/management/pages/index.tsx

```diff
@@ -79,6 +81,13 @@ const EventFilterTelemetry = () => (
   </TrackApplicationView>
 );
 
+const HostIsolationExceptionsTelemetry = () => (
+  <TrackApplicationView viewId={SecurityPageName.hostIsolationExceptions}>
+    <SpyRoute pageName={SecurityPageName.administration} />
```

Comment:
What does this `SpyRoute` thing do? 🤔

Reply:
it keeps an eye on which page is currently active and then it is later used to show the current item in blue, like this:

![image](https://user-images.githubusercontent.com/227916/134924791-f68dfbf9-3c4d-41ec-bb6e-f130a3aeea2b.png)

if you remove the spy, it never turns blue :D

Reply:
Ah! I wonder why don't the other links use the `SpyRoute`